homeassistant/components/bond/light.py

```diff
@@ -29,9 +33,15 @@ async def async_setup_entry(
         for device in devices
         if device.type == DeviceTypes.CEILING_FAN and device.supports_light()
     ]
-
     async_add_entities(lights, True)
 
+    fireplaces = [
+        BondFireplace(hub, device)
+        for device in devices
+        if device.type == DeviceTypes.FIREPLACE
+    ]
+    async_add_entities(fireplaces, True)
```

Comment:
We should only call `async_add_entities` once with all entities.

Reply:
Thank you Martin, fixed here: https://github.com/home-assistant/core/pull/37921